apiary.apib

```diff
@@ -2910,6 +2910,77 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Project items [/projectItems]
+
+Project items are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
```

Comment:
Can we check with product marketing for some copy here?

Reply:
Checked, they (Niels, Benny, Bram) want to keep this the same as it is now.